birdhouse/optional-components/prometheus-longterm-metrics/config/monitoring/prometheus.rules

```diff
@@ -0,0 +1,15 @@
+groups:
+    - name: longterm-metrics-hourly
+      interval: 1h
+      rules:
+        # percentage of the time, over the last hour, that all CPUs were working
+        # 1 means all CPUs were working all the time, 0 means they were all idle all the time
+        - record: instance:cpu_load:avg_rate1h
```

Comment:
So the 2nd Prometheus-longterm will scrap only these 2 new metrics or it will also scrap all the existing ones?

Reply:
By default, only these two. I suggest we add more rules to this default list in the future though.

If you have other metrics that you want it to scrape for long term storage, you just have to give that rule the label `group: longterm-metrics`.